### PR TITLE
Embcessmod 1300 (File layout, environment guard)

### PR DESCRIPTION
**Pull request by MikeOShay**
Moved existing supplier portal UI code to /supplier-R1
Set up routing (and blank test files) for /supplier-auth and /supplier-public folders
Check server environment variable to prevent new pages from being accessed from Production

### CODE REVIEW

**Comment by MikeOShay** [comment]
@ytqsl This is ready for review. I'm not able to add people to the "Reviewers" section of the PR, all I can do is @ in the comments.

File: suppliers/src/UI/embc-supplier/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -13,22 +14,32 @@ const routes: Routes = [
...
+        loadChildren: () => import('./supplier-public/supplier-public.module').then(m => m.SupplierPublicModule)
+      },
       {
         path: 'maintenance',
         loadChildren: () => import('./maintenance/maintenance.module').then(m => m.MaintenanceModule)
       },
       {
         path: 'submission',
-        loadChildren: () => import('./supplierSubmission/supplierSubmission.module').then(m => m.SupplierSubmissionModule)
+        loadChildren: () => import('./supplier-r1/submission/submission.module').then(m => m.SubmissionModule)
```
Comment (ytqsl) [inline suppliers/src/UI/embc-supplier/src/app/app-routing.module.ts:33]: Not fond of this folder name (supplier-r1)

Comment (ytqsl) [reply suppliers/src/UI/embc-supplier/src/app/app-routing.module.ts:33]: Added @SodhiA1 to review

**Comment by ytqsl** [comment]
Added @SodhiA1 to review

File: suppliers/src/UI/embc-supplier/src/app/app-routing.module.ts (hunk 1 above)
Comment (MikeOShay) [reply suppliers/src/UI/embc-supplier/src/app/app-routing.module.ts:33]: Need to make it clear that these are the live deployed pages, separate from the new development in supplier-auth and supplier-public. I'm open for rename ideas.

**Review by SodhiA1 — APPROVED** [review]

File: suppliers/src/UI/embc-supplier/src/app/app-routing.module.ts (hunk 1 above)
Comment (SodhiA1) [reply suppliers/src/UI/embc-supplier/src/app/app-routing.module.ts:33]: supplier-fire-forget?

Comment (MikeOShay) [reply suppliers/src/UI/embc-supplier/src/app/app-routing.module.ts:33]: I was hoping to keep the new dev folders next to each other alphabetically. That's why they have "supplier-" in the names.

Could just "fire-and-forget" work for now? It won't impact the URL.
I think the final version will be moved into "supplier-public" but until designs are finalized I'd like to keep it separate.

Comment (SodhiA1) [reply suppliers/src/UI/embc-supplier/src/app/app-routing.module.ts:33]: yea for sure

**Comment by MikeOShay** [comment]
@SodhiA1 if this looks good can you merge? I don't have write access.

**Pull request merged**